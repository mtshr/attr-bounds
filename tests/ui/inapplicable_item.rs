use attr_bounds::bounds;

#[bounds(T: Clone)]
mod foo {
    struct Bar;
}

fn main() {}
