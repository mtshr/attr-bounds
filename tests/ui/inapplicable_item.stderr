error: The attribute may only be applied to `struct`s, `enum`s, `union`s, `trait`s, `fn`s, `type`s, and `impl` blocks.
 --> tests/ui/inapplicable_item.rs:3:1
  |
3 | #[bounds(T: Clone)]
  | ^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `bounds` (in Nightly builds, run with -Z macro-backtrace for more info)
